applications/GeoMechanicsApplication/python_scripts/geomechanics_solver.py

```diff
@@ -355,6 +355,12 @@ def _add_water_variables(self):
         self.main_model_part.AddNodalSolutionStepVariable(GeoMechanicsApplication.NORMAL_FLUID_FLUX)
         # Add variables for the water conditions
         self.main_model_part.AddNodalSolutionStepVariable(GeoMechanicsApplication.HYDRAULIC_DISCHARGE)
+        
+        # Add integration \ gauss point values that will likely need extrapolating to node
+        self.main_model_part.AddNodalSolutionStepVariable(GeoMechanicsApplication.HYDRAULIC_HEAD)
+        self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.CAUCHY_STRESS_VECTOR)
+        self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.CAUCHY_STRESS_TENSOR)
+        self.main_model_part.AddNodalSolutionStepVariable(GeoMechanicsApplication.FLUID_FLUX_VECTOR)
```

Comment:
This is a point of discussion: ideally we only add the variables we actually want to interpolate. At this moment, I don't see how. Especially when we do a multistage and want to interpolate new variables later